tools/converter/ds2ballerina/src/main/java/org/wso2/ei/tools/ds2ballerina/util/Util.java

```diff
@@ -57,7 +57,7 @@ public static void createConfigProperties(BLangModelBuilder modelBuilder, DataSe
             for (Map.Entry<String, String> entry : configPropertiesMap.entrySet()) {
                 modelBuilder.createStringLiteral(null, null, entry.getKey());
                 modelBuilder.createStringLiteral(null, null, entry.getValue());
-                modelBuilder.addMapStructKeyValue(null, null);
+              //  modelBuilder.addMapStructKeyValue(null, null);
```

Comment:
I removed most of the unnecessary code comments but kept some of them as those need to be revisited later.